packages/components/src/tab-panel/index.tsx

```diff
@@ -220,6 +220,7 @@ const UnforwardedTabPanel = (
 									showTooltip={ !! tab.icon }
 								/>
 							}
+							accessibleWhenDisabled
```

Comment:
We may to have to do the same for the `Tab` component

diff --git a/packages/components/src/tabs/tab.tsx b/packages/components/src/tabs/tab.tsx
index e1aa85c636..8ab67e1ca9 100644
--- a/packages/components/src/tabs/tab.tsx
+++ b/packages/components/src/tabs/tab.tsx
@@ -26,6 +26,7 @@ export const Tab = forwardRef<
 	const instancedTabId = `${ instanceId }-${ tabId }`;
 	return (
 		<StyledTab
+			accessibleWhenDisabled
 			ref={ ref }
 			store={ store }
 			id={ instancedTabId }

Reply:
Done in 312804105575044a83d6086268ec50cee4da26bb.